src/status_im/ui/screens/profile/events.cljs

```diff
@@ -112,4 +112,5 @@
  (fn [{:keys [db] :as cofx} _]
    (handlers-macro/merge-fx cofx
                             {:db (update db :my-profile/seed assoc :step :finish :error nil :word nil)}
-                            (accounts.utils/account-update {:seed-backed-up? true}))))
+                            (accounts.utils/account-update {:seed-backed-up? true
+                                                            :mnemonic        ""}))))
```

Comment:
Why not use nil?

More generally: can we make this an invariant? I.e. add spec/pre-condition for mnemonic being nil/empty. Maybe wrapped in a `safe-fn` call or something?

Reply:
I suppose this is "" to comply with realm schema, but I would prefer it to be nil and the db layer deal with complying with realm schema

Reply:
Yes, it's because of the db schema. Good point, Eric.